CONTRIBUTING.md

```
@@ -1,12 +1,58 @@
 # Contributing
 
-Before submitting a pull request, make sure you have validated all tests pass as well as ensuring 100% code coverage.
+Follow these steps to set up your environment and ensure your changes meet the project's standards.
 
-To run tests, you must have `pytest`, `pytest-asyncio` and `pytest-cov` installed, then run the following command:
+### Setup
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)